resources/aws/routetable.go

```diff
@@ -112,3 +112,43 @@ func (r *RouteTable) Delete() error {
 func (r RouteTable) ID() string {
 	return r.id
 }
+
+func (r RouteTable) getInternetGateway() (string, error) {
+	resp, err := r.Client.DescribeInternetGateways(&ec2.DescribeInternetGatewaysInput{
+		Filters: []*ec2.Filter{
+			&ec2.Filter{
+				Name: aws.String("attachment.vpc-id"),
```

Comment:
I added a couple comments. I don't think moving the magic string to a constant helps readability in this case, but let me know if you disagree.